### PR TITLE
CID-2874: Support .yaml and .yml extensions for manifest file

### PR DESCRIPTION
**Pull request by mohamedlajmileanix**
## 🛠 Changes made
Enhance the functionality of the GitHub agent by adding support for fetching and processing YAML and YML manifest files from repositories. This includes updating the GraphQL queries to handle these file types and ensuring that the service can correctly parse and return the content of these manifest files.

## ✨ Type of change
- [x] New feature (non-breaking change which adds functionality)

## 🧪 How Has This Been Tested?
- [x] should send updates for yaml manifest file
- [x] should send updates for yml manifest file

## 🏎 Checklist:
- [x] My code follows the style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have made corresponding changes to the documentation (README.md)
- [x] My commit message clearly reflects the changes made
- [x] Assigned the appropriate labels (version, PR type, etc.)

### CODE REVIEW

File: src/main/kotlin/net/leanix/githubagent/services/WebhookEventService.kt [hunk 1]
```diff
@@ -47,39 +49,67 @@ class WebhookEventService(
...
+                installationToken
+            )
+            handleManifestFileChanges(
+                "${gitHubEnterpriseProperties.manifestFileDirectory}${ManifestFileName.YML.fileName}",
+                headCommit,
+                repositoryFullName,
+                organizationName,
+                repositoryName,
+                installationToken
+            )
```
Comment (ahmed-ali-55) [inline src/main/kotlin/net/leanix/githubagent/services/WebhookEventService.kt:67]: If `.yaml` is present, we should just ignore `.yml`, shouldn't we?

Comment (henriq-amaral-leanix) [reply src/main/kotlin/net/leanix/githubagent/services/WebhookEventService.kt:67]: It's not clear that now the backend will handle that.
I understood, that now we'll send only one file.
You cannot start to send 2 files and expected that the backend process them.

Comment (alfredo-mfaria) [reply src/main/kotlin/net/leanix/githubagent/services/WebhookEventService.kt:67]: Hi Amin, I understood your reasoning and you are correct if we wanted to cover all edge cases.
But this one is covered by the sentence you added in the readme - "If both .yaml and .yml files exist in a repository, only the .yaml file will be used"

As long as we make this transparent to the user, I don't think we should tackle this scenario.

File: src/main/kotlin/net/leanix/githubagent/services/WebhookEventService.kt [hunk 2]
```diff
@@ -47,39 +49,67 @@ class WebhookEventService(
...
+        installationToken: String
+    ) {
+        when (manifestFilePath) {
+            in headCommit.added, in headCommit.modified -> {
+                val action = when (manifestFilePath) {
+                    in headCommit.added -> ManifestFileAction.ADDED
+                    else -> ManifestFileAction.MODIFIED
+                }
+                logger.info("Manifest file $action in repository $repositoryFullName")
+                val fileContent = gitHubGraphQLService.getManifestFileContent(
```
Comment (alfredo-mfaria) [inline src/main/kotlin/net/leanix/githubagent/services/WebhookEventService.kt:87]: I would solve it here. First checking for yaml and after checking for yml

**Review by henriq-amaral-leanix — APPROVED** [review]

**Pull request merged**